dev-requirements.txt

```
@@ -1,8 +1,8 @@
 # install latest changes in dbt-core + dbt-postgres
 # TODO: how to switch from HEAD to x.y.latest branches after minor releases?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-postgres&subdirectory=plugins/postgres
+git+https://github.com/dbt-labs/dbt-core.git@e8399cd39561b648d460d4fc227e7e0a6eb8c612#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@e8399cd39561b648d460d4fc227e7e0a6eb8c612#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@e8399cd39561b648d460d4fc227e7e0a6eb8c612#egg=dbt-postgres&subdirectory=plugins/postgres
```

@peterallenwebb Could you try updating these to point to the latest commit from Michelle's branch (rather than the original commit):
```suggestion
git+https://github.com/dbt-labs/dbt-core.git@CT-1919/get_column_schema_from_query_macro2#egg=dbt-core&subdirectory=core
git+https://github.com/dbt-labs/dbt-core.git@CT-1919/get_column_schema_from_query_macro2#egg=dbt-tests-adapter&subdirectory=tests/adapter
git+https://github.com/dbt-labs/dbt-core.git@CT-1919/get_column_schema_from_query_macro2#egg=dbt-postgres&subdirectory=plugins/postgres
```